samples/cpp/benchmark_app/main.cpp

```diff
@@ -311,7 +311,7 @@ int main(int argc, char* argv[]) {
                     // set to user defined value
                     if (supported(key)) {
                         device_config[key] = it_device_nstreams->second;
-                    } else if (supported(ov::num_streams.name())) {
+                    } else if (device == "MULTI" || supported(ov::num_streams.name())) {
                         // Use API 2.0 key for streams
```

Comment:
Please cover "-d MULTI:GPU,CPU -nstream GPU:2,CPU:4" case

Reply:
This behaviour was broken by this PR https://github.com/openvinotoolkit/openvino/pull/12279

Reply:
cover "-d MULTI:GPU,CPU -hint none -nstream GPU:2,CPU:4" case